Create samsara-connector.md

amyzhoong opened the pull request:
Documentation for Samsara custom connector

amyzhoong commented:
#sign-off

amyzhoong commented:
> @amyzhoong please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="Samsara"

Court72 commented:
@amyzhoong - Thanks for your pull request. A member of the content team will review it.

Court72 commented:
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

amyzhoong commented:
Hi, I was just wondering what the update is on the merge and if there are any actions I need to take. Thanks!

ptyx507x commented:
@amyzhoong consider not making direct references in your documentation to Power BI unless absolutely necessary. The connector, once certified, will be published in PowerApps, Power BI Desktop & Datasets, Fabric Dataflows, Fabric Datamart, Power BI Dataflows and any other future integration that Power Query might have. 

For example
`Limit on how far back you can retrieve Samsara data in Power BI: Last 6 months`
Seems explicitly a limitation for Power BI. Is that limitation will also happen in other product integrations such as Fabric Dataflows? if yes, please make the correct modifications for your documentation.

The "Web API" is not an authentication for Power Query connectors. Please reference the authentication kinds used in the Power Query SDK and the ones that your connector uses:
https://learn.microsoft.com/en-us/power-query/handling-authentication

amyzhoong commented:
Hi, I have made the changes to the document based on the feedback provided. What steps should I follow to resubmit the markdown file? 
[samsara-connector.md](https://github.com/MicrosoftDocs/powerquery-docs/files/14530779/samsara-connector.md)

amyzhoong commented:
Good afternoon! Just wanted to follow up here and see if there are any updates regarding the PR. I have addressed the feedback Miguel provided and can resubmit the markdown document quickly if there are more changes that need to be made. Thank you!

jonburchel commented:
@amyzhoong can you also update index.md to add your logo and the relevant details for the connector index? You can copy one of the other examples there for the table structure and most of the contents except the logo and name. Otherwise I have the PR ready to merge once that's complete.

Thank you!

JasonWHowell commented:
#reassign:ptyx507x
#assign:jonburchel

amyzhoong commented:
Hi Jon,

I'll update the index.md shortly and let you know when it's done. Thanks!

Best,
Amy

On Mon, Apr 8, 2024 at 4:01 PM Jason Howell ***@***.***>
wrote:

> #reassign:ptyx507x
> #assign:jonburchel
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/MicrosoftDocs/powerquery-docs/pull/490#issuecomment-2043774911>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBZSM65QVJE4DMAGQ2FNPZTY4MOTTAVCNFSM6AAAAABAV3ACPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANBTG43TIOJRGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Amy Zhong *
Associate Product Manager - Developer Ecosystem

samsara.com <https://www.samsara.com/>
<https://www.linkedin.com/company/samsara/>
<https://www.instagram.com/lifeatsamsara/> <https://twitter.com/Samsara>
<https://www.samsara.com/referrals?utm_source=sigcompany&utm_medium=buttons&utm_campaign=ReferralProgram>
<https://www.samsara.com/events?utm_source=sigcompany&utm_medium=buttons&utm_campaign=UpcomingEvents>
<https://kb.samsara.com/hc/en-us?utm_source=sigcompany&utm_medium=buttons&utm_campaign=Support>

ptyx507x commented:
Hi @amyzhoong ! The update to the index.md will be handled by our team. I'll use one of the icons that you submitted with the connector, so it matches exactly what people see in the product.

jonburchel commented:
[heart]         Jon Burchel reacted to your message:
________________________________
From: amyzhoong ***@***.***>
Sent: Wednesday, April 10, 2024 5:49:06 PM
To: MicrosoftDocs/powerquery-docs ***@***.***>
Cc: Jon Burchel ***@***.***>; Assign ***@***.***>
Subject: Re: [MicrosoftDocs/powerquery-docs] Create samsara-connector.md (PR #490)


Hi Jon,

I'll update the index.md shortly and let you know when it's done. Thanks!

Best,
Amy

On Mon, Apr 8, 2024 at 4:01 PM Jason Howell ***@***.***>
wrote:

> #reassign:ptyx507x
> #assign:jonburchel
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/MicrosoftDocs/powerquery-docs/pull/490#issuecomment-2043774911>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBZSM65QVJE4DMAGQ2FNPZTY4MOTTAVCNFSM6AAAAABAV3ACPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANBTG43TIOJRGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


--
*Amy Zhong *
Associate Product Manager - Developer Ecosystem

samsara.com <https://www.samsara.com/>
<https://www.linkedin.com/company/samsara/>
<https://www.instagram.com/lifeatsamsara/> <https://twitter.com/Samsara>
<https://www.samsara.com/referrals?utm_source=sigcompany&utm_medium=buttons&utm_campaign=ReferralProgram>
<https://www.samsara.com/events?utm_source=sigcompany&utm_medium=buttons&utm_campaign=UpcomingEvents>
<https://kb.samsara.com/hc/en-us?utm_source=sigcompany&utm_medium=buttons&utm_campaign=Support>

—
Reply to this email directly, view it on GitHub<https://github.com/MicrosoftDocs/powerquery-docs/pull/490#issuecomment-2048138118>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABIGFM5S3OBTSUILFD42DHLY4V3RFAVCNFSM6AAAAABAV3ACPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANBYGEZTQMJRHA>.
You are receiving this because you were assigned.Message ID: ***@***.***>

amyzhoong commented:
Hi @ptyx507x, sounds good. Just to confirm, does that mean that there are no edits/new files needed from my team at this time?
